### PR TITLE
Update a11y.md

### PR DESCRIPTION
**Pull request by Gorild**
## 📥 Proposed changes

Proofread: Minor rewrite and formatting changes

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: portal/src/texts/profile/a11y.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ## Tilgjengelighet
 
-Universell utforming handler om at vi skal lage løsningene våre med tanke på at brukere kan ha ulik funksjonsevne, midlertidig eller varig. Når vi tenker på å utforme tilgjengelige løsninger, når vi alle målgruppene med én og samme løsning.
+Universell utforming handler om at vi lager løsninger med tanke på at brukere kan ha ulik funksjonsevne. Den kan være  midlertidig eller varig. Eksempler kan være synshemming, nedsatt finmotorikk eller vansker med å skille farger. Når vi planlegger hvordan vi skal utforme løsninger som er tilgjengelig for alle, får vi én løsning som når alle målgrupper.
```
Comment (lfbergee) [inline portal/src/texts/profile/a11y.md]: How about `Universell utforming handler om at vi lager løsninger for alle`, that at least what a11y is to me. Not really the special handling for special needs, but rather common sense that everyone should be able to access our stuff.

File: portal/src/texts/profile/a11y.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+
 En løsning er universelt utformet når den
-gir alle brukere samme muligheter, uansett funksjonsevne
-tilpasser seg brukeren uansett hvor i løsningen de befinner seg
 
-I løsninger som er rettet mot kunder, må Fremtinds produkter følge WCAG-standarden. Den spesifiserer hvordan ulike deler av løsningen skal være best mulig tilgjengelig for alle. Reglene gjelder ikke på samme måte for interne saksbehandlingsløsninger, men ved å følge gode prinsipper for design og tilgjengelighet, oppnår vi de beste løsningene uansett hvem som skal bruke dem.
+- gir alle brukere samme muligheter til å bruke den, uansett funksjonsevne
+- tilpasser seg brukeren, uansett hvor i løsningen de befinner seg
+
+I løsninger som er rettet mot kunder, må Fremtinds produkter følge WCAG-standarden. Den spesifiserer hvordan ulike deler av løsningen skal være best mulig tilgjengelig for alle. Reglene gjelder ikke på samme måte for interne saksbehandlingsløsninger. Målet vårte er likevel at vi skal følge gode prinsipper for design og tilgjengelighet, slik at vi får de beste løsningene, uansett hvem som skal bruke dem.
```
Comment (lfbergee) [inline portal/src/texts/profile/a11y.md]: Actually we can tighten this up even more, we're required by law to comply with WCAG2. And I'd like us to take our ambission a bit further, not only comply with regulation, but test and deliver great user experiences for screen readers, keyboard users and so on.

Comment (lfbergee) [inline portal/src/texts/profile/a11y.md]: Also, I'd be rather careful with the last part here. Fremtind is an inclusive place, and we make sure our internal application is accessible as well. We might not be required by law to comply with WCAG here, but I think we can do better, actually we can customize based on individual needs to make sure Fremtinds employees also have a great user experience.

File: portal/src/texts/profile/a11y.md (hunk 1 above)
Comment (Gorild) [reply portal/src/texts/profile/a11y.md]: hva med "Universell utforming handler om at vi lager gode løsninger for alle". Men mener du at vi da skal ta ut eksemplene på funksjonsnedsettelser? Når jeg jobber med ulike folk her på huset, er det ikke bevisste på at tekst i lenker for eksempel skal kunne leses opp av en skjermleser, så da kan det være viktig med eksempler i Jøkul. Hva tror du?

File: portal/src/texts/profile/a11y.md (hunk 2 above)
Comment (lfbergee) [inline portal/src/texts/profile/a11y.md]: If we keep text as is, `Målet vårte*` should be `Målet vårt`

Comment (Gorild) [reply portal/src/texts/profile/a11y.md]: Da foreslår jeg at vi tar det ut. Fra "Reglene...". Skal du  eller jeg gjør det?

File: portal/src/texts/profile/a11y.md (hunk 1 above)
Comment (piofinn) [reply portal/src/texts/profile/a11y.md]: Good accessibility makes things easier to use for _everyone_, not just those with disabilities. We should definitely make people aware that people with disabilities use our services, but the focus should be that we make the solutions _better_ (just plain better) by having good accessibility 😊

Comment (piofinn) [reply portal/src/texts/profile/a11y.md]: Maybe something like
`Universell utforming handler om å lage løsninger som er lette å bruke for alle. Våre brukere kan ha ulik funksjonsevne. Den kan være midlertidig eller varig nedsatt. Eksempler er synshemming, nedsatt finmotorikk, og nedsatte kognitive evner. Når vi utformer og utvikler løsninger som er tilgjengelige for disse brukerne, får vi løsninger som er bedre for alle målgrupper.`

File: portal/src/texts/profile/a11y.md (hunk 2 above)
Comment (piofinn) [reply portal/src/texts/profile/a11y.md]: @Gorild You should make any changes to the pull request yourself :)

File: portal/src/texts/profile/a11y.md (hunk 1 above)
Comment (Gorild) [reply portal/src/texts/profile/a11y.md]: Kjøpt @piofinn :-)

File: portal/src/texts/profile/a11y.md (hunk 2 above)
Comment (Gorild) [reply portal/src/texts/profile/a11y.md]: Text has been changed, so vårte is "bårte" :-)

Comment (Gorild) [reply portal/src/texts/profile/a11y.md]: Rewritten and committed.

File: portal/src/texts/profile/a11y.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
+
 En løsning er universelt utformet når den
-gir alle brukere samme muligheter, uansett funksjonsevne
-tilpasser seg brukeren uansett hvor i løsningen de befinner seg
 
-I løsninger som er rettet mot kunder, må Fremtinds produkter følge WCAG-standarden. Den spesifiserer hvordan ulike deler av løsningen skal være best mulig tilgjengelig for alle. Reglene gjelder ikke på samme måte for interne saksbehandlingsløsninger, men ved å følge gode prinsipper for design og tilgjengelighet, oppnår vi de beste løsningene uansett hvem som skal bruke dem.
+- gir alle brukere samme muligheter til å bruke den, uansett funksjonsevne
+- tilpasser seg brukeren, uansett hvor i løsningen de befinner seg
+
+Vi følger WCAG-standarden for alle løsninger som skal brukes av kunder. Det samme kravet gjelder ikke for interne saksbehandlingsløsninger, men vi tenker selvsagt på universell utforming og ønsker å lage de aller beste løsningene for alle, uansett hvem som skal bruke dem.
```
Comment (lfbergee) [inline portal/src/texts/profile/a11y.md]: My comment from earlier still applies in this edition. I think some text about how Fremtind is inclusive and how we'll make sure our internal applications is great for everyone, or just drop it here and keep those requirements internal. To me this sentence reads a bit negative.

Comment (Gorild) [reply portal/src/texts/profile/a11y.md]: Please confer with @piofinn  and agree on how to phrase this, and I can proof it at the end.

File: portal/src/texts/profile/a11y.md (hunk 2 above)
Comment (Gorild) [reply portal/src/texts/profile/a11y.md]: Updated with @piofinn suggestion and added a sentence about also describing accessibility for each profile element.

File: portal/src/texts/profile/a11y.md (hunk 3 above)
Comment (Gorild) [reply portal/src/texts/profile/a11y.md]: Updated with @piofinn suggestion and added a sentence about also describing accessibility for each profile element.

**Pull request closed without merging**